media-gfx/gimp/gimp-2.10.14.ebuild

```diff
@@ -0,0 +1,230 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+GNOME2_EAUTORECONF=yes
+
+inherit eapi7-ver virtualx autotools gnome2 python-single-r1
+
+DESCRIPTION="GNU Image Manipulation Program"
+HOMEPAGE="https://www.gimp.org/"
+SRC_URI="mirror://gimp/v$(ver_cut 1-2)/${P}.tar.bz2"
+LICENSE="GPL-3 LGPL-3"
+SLOT="2"
+KEYWORDS="~amd64"
+
+LANGS="am ar ast az be bg br ca ca@valencia cs csb da de dz el en_CA en_GB eo es et eu fa fi fr ga gl gu he hi hr hu id is it ja ka kk km kn ko lt lv mk ml ms my nb nds ne nl nn oc pa pl pt pt_BR ro ru rw si sk sl sr sr@latin sv ta te th tr tt uk vi xh yi zh_CN zh_HK zh_TW"
+
+IUSE="aalib alsa altivec aqua debug doc gnome heif jpeg2k mng postscript python openexr udev unwind vector-icons webp wmf xpm cpu_flags_x86_sse cpu_flags_x86_mmx"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-arch/bzip2
+	>=app-arch/xz-utils-5.0.0
+	>=app-text/poppler-0.50[cairo]
+	>=app-text/poppler-data-0.4.7
+	>=dev-libs/atk-2.2.0
+	>=dev-libs/glib-2.56.0:2
+	dev-libs/libxml2
+	dev-libs/libxslt
+	>=gnome-base/librsvg-2.40.6:2
+	>=media-gfx/mypaint-brushes-1.3.0
+	>=media-libs/babl-0.1.72
+	>=media-libs/fontconfig-2.12.4
+	>=media-libs/freetype-2.1.7
+	>=media-libs/gegl-0.4.18:0.4[cairo]
+	>=media-libs/gexiv2-0.10.6
+	>=media-libs/harfbuzz-0.9.19
+	>=media-libs/lcms-2.8:2
+	>=media-libs/libmypaint-1.3.0:=
+	>=media-libs/libpng-1.6.25:0=
+	>=media-libs/tiff-3.5.7:0
+	net-libs/glib-networking[ssl]
+	sys-libs/zlib
+	virtual/jpeg:0
+	>=x11-libs/cairo-1.12.2
+	>=x11-libs/gtk+-2.24.32:2
+	>=x11-libs/gdk-pixbuf-2.31:2
+	>=x11-libs/pango-1.29.4
+	x11-libs/libXcursor
+	x11-themes/hicolor-icon-theme
+	aalib? ( media-libs/aalib )
+	alsa? ( >=media-libs/alsa-lib-1.0.0 )
+	aqua? ( >=x11-libs/gtk-mac-integration-2.0.0 )
+	gnome? ( gnome-base/gvfs )
+	heif? ( >=media-libs/libheif-1.3.2:= )
+	jpeg2k? ( >=media-libs/openjpeg-2.1.0:2= )
+	mng? ( media-libs/libmng:= )
+	openexr? ( >=media-libs/openexr-1.6.1:= )
+	postscript? ( app-text/ghostscript-gpl )
+	python?	(
+		${PYTHON_DEPS}
+		>=dev-python/pycairo-1.0.2[${PYTHON_USEDEP}]
+		>=dev-python/pygtk-2.10.4:2[${PYTHON_USEDEP}]
+	)
+	udev? ( dev-libs/libgudev:= )
+	unwind? ( >=sys-libs/libunwind-1.1.0:= )
+	webp? ( >=media-libs/libwebp-0.6.0:= )
+	wmf? ( >=media-libs/libwmf-0.2.8 )
+	xpm? ( x11-libs/libXpm )
+	"
+
+DEPEND="
+	${RDEPEND}
+	>=dev-lang/perl-5.10.0
+	dev-libs/appstream-glib
+	dev-util/gtk-doc-am
+	dev-util/gtk-update-icon-cache
+	>=dev-util/intltool-0.40.1
+	sys-apps/findutils
+	>=sys-devel/automake-1.11
+	>=sys-devel/gettext-0.19
+	>=sys-devel/libtool-2.2
+	virtual/pkgconfig
+	doc? ( >=dev-util/gtk-doc-1.0 )
+	"
+
+DOCS="AUTHORS HACKING NEWS README README.i18n"
+
+# Bugs 685210 (and duplicate 691070)
+PATCHES="${FILESDIR}/${PN}-$(ver_cut 1-2)_fix_test-appdata.patch"
```

Comment:
there's no reason to use `ver_cut` here, just make it `2.10`
also should be an array instead.

Reply:
Done